README.md

```diff
@@ -1,6 +1,9 @@
+[![Technologiestiftung Berlin](https://img.shields.io/badge/Built%20with%20%E2%9D%A4%EF%B8%8F-at%20Technologiestiftung%20Berlin-blue)](https://www.technologiestiftung-berlin.de)
+
 # Innovationsdaten
 
-Daten aus 10 Jahre Innovationserhebung ab Innovationserhebung 2013
+Daten aus 10 Jahre Innovationserhebung seit 2012
```

Comment:
Corrected the information, but this should be in English. @marcoka2000 some suggestion about a slightly longer description?

Reply:
Will come up with a suggestion later today 👨🏻‍💻

Reply:
Looks good to me, thanks!